### PR TITLE
New package: kkclue.GarfieldPlus version 1.01

### PR DESCRIPTION
**Pull request by manOnWebs**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/141679)

### CODE REVIEW

**Comment by manOnWebs** [comment]
@microsoft-github-policy-service agree

File: manifests/k/kkclue/GarfieldPlus/1.01/kkclue.GarfieldPlus.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: kkclue.GarfieldPlus
+PackageVersion: "1.01"
+PackageLocale: en-US
+Publisher: KKClue
+PackageName: Garfield+
```
Comment (stephengillie) [inline manifests/k/kkclue/GarfieldPlus/1.01/kkclue.GarfieldPlus.locale.en-US.yaml:8]:
```suggestion
PackageName: Garfield+
PackageUrl: https://planetclue.com/garfieldplus/
```

File: manifests/k/kkclue/GarfieldPlus/1.01/kkclue.GarfieldPlus.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+PackageIdentifier: kkclue.GarfieldPlus
+PackageVersion: "1.01"
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: garfield.exe
+  PortableCommandAlias: garfield
+Installers:
+- InstallerUrl: https://archive.org/download/garfield-plus/GarfieldPlus-v1.01.zip
```
Comment (stephengillie) [inline manifests/k/kkclue/GarfieldPlus/1.01/kkclue.GarfieldPlus.installer.yaml:12]:
```suggestion
- InstallerUrl: https://pixeldrain.com/api/filesystem/UgVhcrVc/GarfieldPlus-v1.01.zip?attach
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @manOnWebs,

I found what appears to be the PackageUrl for this software package. But this points at a different host - should this InstallerUrl be used instead? If not, can you provide a PackageUrl that links to this InstallerUrl?

**Comment by stephengillie** [comment]
Completing validation.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**